testrail-integration/src/main/java/org/folio/test/services/TestRailService.java

```diff
@@ -27,51 +26,24 @@ public class TestRailService {
 
   private final TestRailDao testRailDao;
   private final TestRailClient testRailClient;
-  private final HashSet<Integer> cachedCaseIds;
 
   public TestRailService() {
     this.testRailDao = new TestRailDao();
     this.testRailClient = new TestRailClient();
-    this.cachedCaseIds = new HashSet<>();
-  }
-
-  public void cacheCaseIds(Integer runId) {
-    var requestOffset = 0;
-    var requestLimit = 50;
-    while (true) {
-      logger.info("cacheCaseIds:: Retrieving tests, offset: {}, limit: {}", requestOffset, requestLimit);
-      var responsePayload = testRailDao.getTests(testRailClient, runId, requestOffset * requestLimit, requestLimit);
-      if (responsePayload == null) {
-        break;
-      }
-
-      var tests = responsePayload.tests();
-      if (tests.isEmpty()) {
-        break;
-      }
-      var caseIds = tests.stream()
-        .filter(Objects::nonNull)
-        .map(Test::caseId)
-        .toList();
-      cachedCaseIds.addAll(caseIds);
-
-      if (responsePayload.links().next() == null) {
-        break;
-      }
-      logger.info("cacheCaseIds:: Offset: {}, next: {}", responsePayload.offset(), responsePayload.links().next());
-      requestOffset++;
-    }
-
-    logger.info("cacheCaseIds:: Cached {} case ids", cachedCaseIds.size());
   }
 
   public void createResults(Integer runId, Map<String, Results> results) {
+    var caseIds = SharedInstanceInitializer.getSharedInstance().getCaseIds();
```

Comment:
name a bit confuses with shared instances functionality that we implemented recently.
Can rename to SharedCacheInitializer.getSharedCache()